go.mod

```diff
@@ -2,6 +2,8 @@ module sigs.k8s.io/cluster-api-provider-vsphere
 
 go 1.17
 
+replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.1.3
```

Comment:
I am not sure if we really need this. Why can't we update our cluster-api version than doing this?

Reply:
`v1.1.3` is the latest release right now. After some investigation, I think this issue was introduced when we upgraded to `v1.1.3`  and removed the `replace` directives with bf4a27f7086d95ea437b3b142ebd435f94dea4ce.